README.md

```diff
@@ -11,7 +11,7 @@ You can install the PHP SDK with Composer, either run `composer require microsof
 ```
 {
     "require": {
-        "microsoft/microsoft-graph": "^1.43.0"
+        "microsoft/microsoft-graph": "^2.1.0-RC0"
```

Comment:
Interesting. This is caused by the workflow pulling the minor bumping the latest available version on Packagist.
Manually making a commit to change this and disable the version bump until v2 is stable

Reply:
PR in #697